src/components/features/MessageDetail/MessageDetailHeader/index.tsx

```diff
@@ -34,15 +39,15 @@ const MessageDetailHeader = ({ profileImg, treeId, senderName, isLike, messageId
 
 			<S.MessageMenu>
 				{isLike ? (
-					<S.LikeButton onClick={onUpdateFavoriteMessage}>
+					<S.LikeButton onClick={() => updateFavoriteMutate()}>
 						<img src={LikeIcon} alt="" />
 					</S.LikeButton>
 				) : (
-					<S.LikeButton onClick={onUpdateFavoriteMessage}>
+					<S.LikeButton onClick={() => updateFavoriteMutate()}>
 						<img src={StarIcon} alt="" style={{ width: '15px', height: '15px' }} />
 					</S.LikeButton>
```

Comment:
```suggestion
					<S.LikeButton onClick={() => updateFavoriteMutate()}>
						<img src={isLike ? LikeIcon : StarIcon} alt="" />
					</S.LikeButton>
```

이후 리팩토링 하실 때 인라인 스타일 대신 emotion 을 사용해주시면 좋을 것 같습니다!
그리고 스타일 상 가능하다면 이렇게 안에서 분기처리 하는 것도 간결해보이네요!